pages/form-and-list.js

```diff
@@ -0,0 +1,200 @@
+import { useState } from "react";
+import { nanoid } from "nanoid";
+import styled from "styled-components";
+
+export default function FormAndList() {
+  const [rangeValue, setRangeValue] = useState(127.5);
+  const [allEntries, setallEntries] = useState([]);
+
+  const handleRangeChange = (event) => {
+    setRangeValue(event.target.value);
+  };
+
+  function onSubmit(event) {
+    event.preventDefault();
+    const submittedForm = {
+      id: nanoid(),
+      experiences: [
+        { anger: event.target.anger.checked },
+        { enjoyment: event.target.enjoyment.checked },
+        { fear: event.target.fear.checked },
+        { disgust: event.target.disgust.checked },
+        { sadness: event.target.sadness.checked },
+      ],
+      text: event.target.text.value,
+      slider: rangeValue,
+
+      date: new Date().toLocaleString(),
+    };
+
+    setallEntries([...allEntries, submittedForm]);
+    // console.log([...allEntries, submittedForm]);
+    event.target.reset();
+  }
+
+  function Intensity({ intensity }) {
+    if (intensity <= 51) {
+      return <span>very sad</span>;
+    }
+    if (intensity >= 51 && intensity <= 102) {
+      return <span>sad</span>;
+    }
+    if (intensity >= 102 && intensity <= 153) {
+      return <span>pleasent</span>;
+    }
+    if (intensity >= 153 && intensity > 206) {
+      return <span>extremely good</span>;
+    }
+  }
+
+  return (
+    <div
+      style={{
+        display: "flex",
+        justifyContent: "start",
+        alignItems: "center",
+        flexDirection: "column",
+        backgroundColor: "black",
+        color: "white",
+        minHeight: "100vh",
+      }}
```

Comment:
no inline styling please :) only styled components!
you can create a Container Styled component here